README.md

```diff
@@ -98,6 +108,45 @@ Use the following settings to update cell execution time for a notification and
 
 ![notification](https://user-images.githubusercontent.com/3497137/111881088-01db5200-897d-11eb-8faa-4701cabfcde4.gif)
 
+## (Option) Notifications using `ntfy`
+
+You can recieve notifications via `ntfy`.
+
+**ntfy 2.7.0 documentation** https://ntfy.readthedocs.io/en/latest/
+
+> ntfy brings notification to your shell. It can automatically provide desktop notifications when long running commands finish or it can send push notifications to your phone when a specific command finishes.
+
+### How to work
+
+Install `ntfy`.
+
+```console
+$ pip install ntfy
+```
+Configure ntfy in accodance with [the ntfy official document](https://ntfy.readthedocs.io/en/latest/#configuring-ntfy)
+
+```console
+$ vim ~/.config/ntfy/ntfy.yml
+```
+
+```yaml
+backends:
+    - pushover
+pushover:
+    user_key: hunter2
```

Comment:
I'm not familiar with `ntfy`. Do users have to create their own user_key to enable `ntfy`? Or will they use the same `hunter2` user_key?

Reply:
@mwakaba2 
No, intergration with pushover is just only option, and `hunter2` implies `YOUR_PUSHOVER_USER_KEY`. The code on `How to enable notifications via ntfy` subsection are example codes.

It seems to be poorly-written.... so I'll also update sentenses on this subsection.

Reply:
Sounds great! Thanks for the clarification and doc update!